exercises/gigasecond/.meta/template.j2

```diff
@@ -0,0 +1,17 @@
+{%- import "generator_macros.j2" as macros with context -%}
+from datetime import datetime
+{{ macros.header() }}
+
+class {{ exercise | camel_case }}Test(unittest.TestCase):
+    {% for supercase in cases -%}
+    {% for case in supercase["cases"] -%}
+    {% set input = case["input"]["moment"] -%}
+    {% set expected = case["expected"] -%}
+    def test_{{ case["description"] | to_snake }}(self):
+        self.assertEqual({{ case["property"] }}(
+            datetime.fromisoformat("{{ input }}")
```

Comment:
`datetime.fromisoformat` was [introduced in Python3.7](https://docs.python.org/3/library/datetime.html#datetime.date.fromisoformat). We'll need an alternate solution.

Perhaps to avoid requiring students to have `dateutil`, we could add it to the requirements for the generator, and use our template plugins feature to convert the ISO datetime strings from the spec into a simple datetime constructor?

Reply:
Ahh hell, didn't notice it was that new ... grr. The `strptime` approach isn't going to work because of the mixed formats they're passing in from the problem spec.

Yeah ... let's add it to the generator and do the conversion in the templates.

Reply:
Pushed changes use dateutil ... however must be merged on top of #2134.